### ISSUE TITLE
vsphere-automation-vcenter gem ships gems

### ISSUE THREAD

**Issue by tas50** [chef/chef-workstation#620]
It looks like they're accidentally packaging the pkg dir that includes a 2.3 meg gem of the previous release. We should work with them to remove this so we can drop 2.3 megs from our install size

**Comment by tas50** [comment]
vsphere-automation-cis is the same way

**Comment by tas50** [comment]
```
./vsphere-automation-content-0.4.5/pkg/vsphere-automation-content-0.4.3.gem
./vsphere-automation-content-0.4.5/pkg/vsphere-automation-content-0.4.2.gem
./vsphere-automation-content-0.4.5/pkg/vsphere-automation-content-0.4.1.gem
./vsphere-automation-content-0.4.5/pkg/vsphere-automation-content-0.4.4.gem
./vsphere-automation-vcenter-0.4.5/pkg/vsphere-automation-vcenter-0.4.4.gem
./vsphere-automation-vcenter-0.4.5/pkg/vsphere-automation-vcenter-0.4.1.gem
./vsphere-automation-vcenter-0.4.5/pkg/vsphere-automation-vcenter-0.4.3.gem
./vsphere-automation-vcenter-0.4.5/pkg/vsphere-automation-vcenter-0.4.2.gem
./vsphere-automation-vapi-0.4.5/pkg/vsphere-automation-vapi-0.4.3.gem
./vsphere-automation-vapi-0.4.5/pkg/vsphere-automation-vapi-0.4.2.gem
./vsphere-automation-vapi-0.4.5/pkg/vsphere-automation-vapi-0.4.1.gem
./vsphere-automation-vapi-0.4.5/pkg/vsphere-automation-vapi-0.4.4.gem
./vsphere-automation-cis-0.4.5/pkg/vsphere-automation-cis-0.4.5.gem
./vsphere-automation-cis-0.4.5/pkg/vsphere-automation-cis-0.4.4.gem
./vsphere-automation-cis-0.4.5/pkg/vsphere-automation-cis-0.4.3.gem
./vsphere-automation-cis-0.4.5/pkg/vsphere-automation-cis-0.4.2.gem
./vsphere-automation-cis-0.4.5/pkg/vsphere-automation-cis-0.4.1.gem
./vsphere-automation-sdk-0.4.5/pkg/vsphere-automation-sdk-0.4.4.gem
./vsphere-automation-sdk-0.4.5/pkg/vsphere-automation-sdk-0.4.1.gem
./vsphere-automation-sdk-0.4.5/pkg/vsphere-automation-sdk-0.4.0.gem
./vsphere-automation-sdk-0.4.5/pkg/vsphere-automation-sdk-0.4.2.gem
./vsphere-automation-sdk-0.4.5/pkg/vsphere-automation-sdk-0.4.3.gem
./vsphere-automation-appliance-0.4.5/pkg/vsphere-automation-appliance-0.4.4.gem
./vsphere-automation-appliance-0.4.5/pkg/vsphere-automation-appliance-0.4.2.gem
./vsphere-automation-appliance-0.4.5/pkg/vsphere-automation-appliance-0.4.3.gem
```

**Comment by tas50** [comment]
9.2mb total for these gems

**Comment by tas50** [comment]
Resolved by https://github.com/vmware/vsphere-automation-sdk-ruby/pull/64

**Comment by tas50** [comment]
The PR to the upstream vmware repo also removes the docs directory from each gem, which drops the total install size / file count as follows:

- 38.5mb on disk -> 11.5mb on disk
- 4331 files -> 1423 files

**Comment by tas50** [comment]
This shipped in 0.4.6 which has been pulled into workstation
